core/create/src/index.ts

```diff
@@ -44,9 +44,19 @@ function installHooks(logger: typeof winstonLogger) {
   }).default(logger)
 }
 
-function hasToolKitConflicts(error: unknown) {
-  // we need to import hasToolkitConflicts from the app itself instead of npx or else hasToolkitConflicts and ToolKitConflictError will come from npx but error will come from the app level, leading to error failing the instanceof ToolKitConflictError check
-  return (importCwd('@dotcom-tool-kit/error') as typeof ToolkitErrorModule).hasToolKitConflicts(error)
+function hasToolKitConflicts(error: unknown): boolean {
+  try {
+    // we need to import hasToolkitConflicts from the app itself instead of npx
+    // or else hasToolkitConflicts and ToolKitConflictError will come from npx
+    // but the error will come from the app level, leading to the error failing
+    // the instanceof ToolKitConflictError check
+    const errorModule = importCwd('@dotcom-tool-kit/error') as typeof ToolkitErrorModule
+    return errorModule.hasToolKitConflicts(error)
+  } catch {
+    // if the error package isn't found then that probably means npm install
+    // has failed and we should be logging out that error
+    return false
```

Comment:
Returning false here will [invoke this line](https://github.com/Financial-Times/dotcom-tool-kit/blob/f67f46fb83074ffa602cbfa6e2572cd424a7f90c/core/create/src/index.ts#L651) and therefore the winston will log the error and the process will exit?